src/Uno.UI/UI/Xaml/RectHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Diagnostics.Contracts;
+using Windows.Foundation;
+
+namespace Windows.UI.Xaml
+{
+	public partial class RectHelper
```

Comment:
Would be nice to add some comments in this class :)

Reply:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.recthelper